packages/theme/pages/Checkout/Billing.vue

```diff
@@ -5,8 +5,30 @@
       :title="$t('Billing')"
       class="sf-heading--left sf-heading--no-underline title"
     />
+    <SfAddressPicker
```

Comment:
I think the logic you added to Billing.vue and Shipping.vue is pretty much the same.
Could you extract it to a component, like `Checkout/AddressPicker` or something like that?